Formula/klavaro.rb

```diff
@@ -21,6 +21,7 @@ class Klavaro < Formula
   depends_on "intltool" => :build
   depends_on "pkg-config" => :build
   depends_on "gtk+3"
+  depends_on "adwaita-icon-theme"
```

Comment:
```suggestion
  depends_on "adwaita-icon-theme"
  depends_on "gtk+3"
```

Reply:
Its not clear why you have put the theme dependency after gtk+3. I would have thought that as when manually installing you would install gtk+3 and then install the icon theme. Indeed I considered adding this change to the gtk+3 formula, but the style for other formulas seems to be to include it in the required application

Reply:
The `adwaita-icon-theme` formula does not depend on or require `gtk+3`, and runtime dependencies are ordered alphabetically.

As-is your commit will fail with a style error:
```
➜ brew style klavaro
Formula/klavaro.rb:24:3: C: [Correctable] dependency "adwaita-icon-theme" (line 24) should be put before dependency "gtk+3" (line 23)
  depends_on "adwaita-icon-theme"
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```